python/plugins/db_manager/db_plugins/postgis/connector.py

```diff
@@ -719,19 +719,19 @@ def deleteTable(self, table):
         self._execute_and_commit(sql)
 
     def emptyTable(self, table):
-        """ delete all rows from table """
+        """Deletes all rows from table """
         sql = u"TRUNCATE %s" % self.quoteId(table)
         self._execute_and_commit(sql)
 
-    def renameTable(self, table, new_table):
-        """ rename a table in database """
+    defRenamesTable(self, table, new_table):
```

Comment:
It was correct before I guess too

Reply:
Yeah, I noticed it myself, see my last commit